ADAL/tests/ADTestCase.m

```diff
@@ -40,6 +40,8 @@ - (void)setUp
 
 - (void)tearDown
 {
+    XCTAssertTrue([ADTestURLSession noResponsesLeft]);
```

Comment:
@RPangrle I think tearDown is not a good place for a XCTAssert, because if it fails, Xcode does't highlight it inside a test and it is not obvious that assert failed during tearDown phase. I would say it is better to avoid XCTAssert in tearDown and put it  always inside a test.